src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -173,16 +173,27 @@ export function ConfirmationModalContent({
   bottomContent,
   onDismiss,
   topContent,
+  chainId,
 }: {
   title: ReactNode
   onDismiss: () => void
   topContent: () => ReactNode
   bottomContent?: () => ReactNode | undefined
+  chainId?: SupportedL2ChainId | undefined
 }) {
+  const info = getChainInfo(chainId)
   return (
     <Wrapper>
       <AutoColumn gap="sm">
         <Row>
+          {chainId && (
+            <Badge>
+              <RowFixed data-testid="confirmation-modal-chain-icon">
+                <StyledLogo src={info?.logoUrl} style={{ margin: '0 8px 0 0' }} />
```

Comment:
Is this right margin? What is it needed for? Also, prefer Styled Components if absolutely necessary.